src/poker/game/GamePlay.java

```diff
@@ -0,0 +1,545 @@
+package poker.game;
+
+import poker.*;
+
+import java.lang.ref.Cleaner;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class GamePlay {
+    public Deck deck;
+    public MakeBoardAndCards boardAndCards;
+    public HandsAndBoard result;
+    public Player player1;
+    public Player player2;
+    GameState gameState;
+    List<Player> playerList;
+    public double smallBlind = 0.5;
+    public double bigBlind = 1;
+    double maxValueRaiseBigBlind = 0;
+    double maxValueRaiseSmallBlind = 0;
+    boolean isFirstRaiseBigBlind = true;
+    boolean isFirstRaiseSmallBlind = true;
+    boolean smallBlindCallPreFlop = true;
+    public EquityEvaluator equityEvaluator;
+    public ResultHandOut resultHandOut;
+    public GameResult gameResult;
+    boolean firstHand = true;
+
+    public GamePlay() {
+        this.deck = new Deck();
+        this.boardAndCards = new MakeBoardAndCards(deck);
+        this.result = boardAndCards.makeHandsBoard();
+        this.player1 = new Player(100, 0, result.getHand1(), "player1");
+        this.player2 = new Player(100, 0, result.getHand2(), "player2");
+        this.gameState = new GameState(GamePhase.PREFLOP, 0);
+        playerList = new ArrayList<>();
+        playerList.add(player1);
+        playerList.add(player2);
+        this.equityEvaluator = new EquityEvaluator(deck, result.getHand1(), result.getHand2(), result.getBoard());
+        this.gameResult = new GameResult();
+    }
+
+    // wykryc kto wygra na river (bez all inow) udpate staka i rozpoczęcie nowego handa, potem uprzątnać to co się da
+    // commit i push wysłać linka zadanie3 do git huba
+    public void gameProgress() {
+        int i = 0;
+        while (playerList.size() > 1) {
+            if (firstHand) {
+            } else {
+                try {
+                    Thread.sleep(4000);
+                } catch (InterruptedException e) {
+                    e.printStackTrace();
+                }
+            }
```

Comment:
to powinno byc na samym koncu w petli dzieki temy nie bedziesz potrzebowal if i zmiennej globalnej firstHand bo wtedy zawsze chcesz wywolac Thread.sleep(4000);